StratisMasternodeDashboard-master/Controllers/SidechainNodeController.cs

```diff
@@ -79,22 +79,40 @@ public async Task<IActionResult> ChangeLogLevelAsync(string rule, string level)
         }
 
         [Ajax]
-        [HttpPost] 
+        [HttpPost]
         [Route("vote")]
         public async Task<IActionResult> Vote(Vote vote)
         {
             if (string.IsNullOrEmpty(vote?.Hash))
                 return this.BadRequest("Hash is required");
-            
+
             ApiResponse response = await this.apiRequester.PostRequestAsync(this.defaultEndpointsSettings.SidechainNode, "/api/Voting/schedulevote-whitelisthash", new { hash = vote.Hash });
 
             if (response.IsSuccess) return this.Ok();
             if (response.Content?.errors != null)
             {
                 return this.BadRequest($"Failed to whitelist hash. Reason: {response.Content?.errors[0].message}");
             }
-            
+
             return this.BadRequest($"Failed to whitelist hash. Reason: {response.Content}");
         }
+
+        [HttpPost]
+        [Route("schedulekick")]
+        public async Task<IActionResult> ScheduleKick([FromBody] string pubKey)
+        {
+            if (string.IsNullOrEmpty(pubKey))
+                return this.BadRequest("Member key is required");
+            ApiResponse response = await this.apiRequester.PostRequestAsync(this.defaultEndpointsSettings.SidechainNode, "/api/Voting/schedulevote-kickmember", new { pubkey = pubKey });
+            if (response.IsSuccess)
```

Comment:
You stil need to chnge the whole rest of the method so that it matches this:

```
 if (response.IsSuccess) 
     return this.Ok();

if (response.Content?.errors != null)
     return this.BadRequest($"An error occurred trying to schedule a kick federation member vote:               {response.Content?.errors[0].message}");
                
return this.BadRequest($"An error occurred trying to schedule a kick federation member vote: {response.Content}");
```

Cause in your version if there are Errors `IsSuccess` will be false if there are errors or not so ou first need to check for errors and if that is not null display them, otherwise, display return `response.Content` in the request.